hom/cogs/views.py

```diff
@@ -0,0 +1,344 @@
+            "or delete a group, so please be mindful of who you choose to share it with.\n\n"
+            f"{Constants.FOOTER}"
+        )
+
+        await interaction.response.defer()
+        await utils.create_ticket_for_user(
+            interaction,
+            instructions,
+            f"Group {Constants.ARROW} {button.label}",
+            "https://cdn.discordapp.com/attachments/1125434806411464867/1127741141152960542/image.png",
+        )
+
+    @discord.ui.button(
+        label="Remove me from a group",
+        style=discord.ButtonStyle.blurple,
+        custom_id="persistent_view:group_remove",
+    )
+    async def group_remove(
+        self,
+        interaction: discord.Interaction[commands.Bot],
+        button: discord.ui.Button["SupportGroup"],
+    ) -> None:
+        instructions = (
+            "To remove yourself from a group, please provide us with a screenshot containing:"
+            "\n\n- Your in-game username\n- Your Discord username/ID\n- Today's date\n\n"
+            f"{Constants.FOOTER}"
+        )
+
+        await interaction.response.defer()
+        await utils.create_ticket_for_user(
+            interaction,
+            instructions,
+            f"Group {Constants.ARROW} {button.label}",
+            "https://cdn.discordapp.com/attachments/1125434806411464867/1129178517557481585/image.png",
+        )
+
+    @discord.ui.button(
+        label="Other",
+        style=discord.ButtonStyle.blurple,
+        custom_id="persistent_view:group_other",
+    )
+    async def group_other(
+        self,
+        interaction: discord.Interaction[commands.Bot],
+        button: discord.ui.Button["SupportGroup"],
+    ) -> None:
+        instructions = f"Explain what you require assistance with below.\n\n{Constants.FOOTER}"
+
+        await interaction.response.defer()
+        await utils.create_ticket_for_user(
+            interaction, instructions, f"Group {Constants.ARROW} {button.label}"
+        )
+
+
+class SupportNames(discord.ui.View):
+    def __init__(self) -> None:
+        super().__init__(timeout=None)
+
+    @discord.ui.button(
+        label="Approve a pending name change",
+        style=discord.ButtonStyle.blurple,
+        custom_id="persistent_view:names_approve",
+    )
+    async def names_approve(
+        self,
+        interaction: discord.Interaction[commands.Bot],
+        button: discord.ui.Button["SupportNames"],
+    ) -> None:
+        instructions = (
+            "Some name changes get skipped, as they can’t be auto-approved by our system and "
+            "require manual approval.\n\nIf yours hasn’t been auto-approved, please tell us the "
+            "name change ID and we’ll manually review it for you.\n\nNote: If you’d like to know "
+            "why your name change has been skipped you can visit our beta website (work in "
+            "progress) at https://beta.wiseoldman.net/names and hover your cursor over the your "
```

Comment:
The new site has been released and this text should be changed a little.

Reply:
good catch!